### DISCUSSION TITLE
Proposal: Shorter Documentation Comments without <summary>

### DISCUSSION THREAD

**Discussion by exyi** [dotnet/csharplang#2549] [Language Ideas]
Typical documentation comment in C# has only the `<summary>` section and is usually just one line of text. I don't like the fact, that one-line comment has to have another two dummy lines to make tooling happy.

What about having the summary element optional, like F# has: https://docs.microsoft.com/en-us/dotnet/articles/fsharp/language-reference/xml-documentation?

**Comment by qrli** [comment]
Yeah, you bring it up.

I have been feeling the pain for long. With documentation comments, there is usually much more /// lines than code, and there is so many tags to distract from real doc text. I've thought about using Doxygen instead, but that format won't be picked up by intellisense.

When Visual Studio got the features of `Go to Definition` of a DLL with a generated declaration source file, I really liked the doc comments shown there. No bloated xml format but only the content that I want to read. Really terse. It should have been the doc comment format of C#! It was bad luck that C# was born in the XML hype days.

**Comment by alrz** [comment]
wow. https://anvaka.github.io/common-words/#?lang=cs
